File: LastSemesterInfo/AYDA Laboratorio/AYDA_Examen_1_student_distribution_(A2026)/linux/longest_common_subsequence/cpp/template.cpp
#include <iostream>
#include <vector>
#include <fstream>

using namespace std;

vector<int> longestCommonSubsequence(vector<int> a, vector<int> b)
{
    return vector<int>();
}

int main()
{
    int n, m;
    if (!(cin >> n >> m)) return 0;

    vector<int> a(n), b(m);
    for (int i = 0; i < n; ++i) cin >> a[i];
    for (int i = 0; i < m; ++i) cin >> b[i];

    vector<int> result = longestCommonSubsequence(a, b);

    ofstream fout("solution.txt");
    for (size_t i = 0; i < result.size(); ++i) {
        fout << result[i] << (i == result.size() - 1 ? "" : " ");
    }
    fout << "\n";
    fout.close();
    return 0;
}

File: LastSemesterInfo/AYDA Laboratorio/AYDA_Examen_1_student_distribution_(A2026)/linux/longest_common_subsequence/python/template.py
import sys

def longestCommonSubsequence(a, b):
    return []

if __name__ == '__main__':
    input_data = sys.stdin.read().split()
    if not input_data:
        sys.exit()

    n = int(input_data[0])
    m = int(input_data[1])

    a = [int(x) for x in input_data[2 : 2+n]]
    b = [int(x) for x in input_data[2+n : 2+n+m]]

    result = longestCommonSubsequence(a, b)

    with open('solution.txt', 'w') as f:
        f.write(' '.join(map(str, result)) + '\n')